src/test/resources/tst_manifests/gradle/deps_with_duplicate_different_versions/expected_stack_sbom.json

```diff
@@ -29,18 +29,31 @@
       "purl" : "pkg:maven/log4j/log4j@1.2.17",
       "type" : "library",
       "bom-ref" : "pkg:maven/log4j/log4j@1.2.17"
+    },
+    {
+      "group" : "log4j",
+      "name" : "log4j",
+      "version" : "1.2.14",
+      "purl" : "pkg:maven/log4j/log4j@1.2.14",
+      "type" : "library",
+      "bom-ref" : "pkg:maven/log4j/log4j@1.2.14"
     }
   ],
   "dependencies" : [
     {
       "ref" : "pkg:maven/org.acme.dbaas/postgresql-orm-quarkus@1.0.0-SNAPSHOT",
       "dependsOn" : [
-        "pkg:maven/log4j/log4j@1.2.17"
+        "pkg:maven/log4j/log4j@1.2.17",
+        "pkg:maven/log4j/log4j@1.2.14"
```

Comment:
Hi @olavtar ,

This is incorrect, only one version should be in the jar package, although the dependency specified twice with two different versions in the manifest, the dependency resolution decision ( made by gradle) is to use the more recent version.

Reply:
Done

Reply:
@olavtar Thanks,
Can you just clean/remove *.orig junk files, that somehow got their way into the repo by mistake?
so the build will pass successfully, and i'll approve this one.

Reply:
Done